components/dropdown/script.js

```diff
@@ -310,7 +339,7 @@ class GamefaceDropdown extends CustomElementValidator {
         components.loadResource(this)
             .then((result) => {
                 this.template = result.template;
-                components.renderOnce(this);
+                components.simpleRender(this, '.guic-dropdown-options', this.querySelectorAll('dropdown-option'));
```

Comment:
Think of a better name.

Reply:
components.**renderChildren** or **transferChildren** ?